recipes/libbasisu/all/patches/cmakelist_build_lib.patch

```diff
@@ -96,20 +97,57 @@
  		set(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} -DBASISU_SUPPORT_SSE=1")
  		set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -DBASISU_SUPPORT_SSE=1")
  	else()
-@@ -94,7 +74,6 @@ else()
+@@ -93,64 +74,51 @@ else()
+ 	endif()
  endif()
  
- set(BASISU_SRC_LIST ${COMMON_SRC_LIST} 
+-set(BASISU_SRC_LIST ${COMMON_SRC_LIST} 
 -	basisu_tool.cpp
- 	encoder/basisu_backend.cpp
- 	encoder/basisu_basis_file.cpp
- 	encoder/basisu_comp.cpp
-@@ -115,42 +94,27 @@ set(BASISU_SRC_LIST ${COMMON_SRC_LIST}
- 	encoder/jpgd.cpp
- 	encoder/basisu_kernels_sse.cpp
- 	transcoder/basisu_transcoder.cpp
--	)
-+)
+-	encoder/basisu_backend.cpp
+-	encoder/basisu_basis_file.cpp
+-	encoder/basisu_comp.cpp
+-	encoder/basisu_enc.cpp
+-	encoder/basisu_etc.cpp
+-	encoder/basisu_frontend.cpp
+-	encoder/basisu_global_selector_palette_helpers.cpp
+-	encoder/basisu_gpu_texture.cpp
+-	encoder/basisu_pvrtc1_4.cpp
+-	encoder/basisu_resampler.cpp
+-	encoder/basisu_resample_filters.cpp
+-	encoder/basisu_ssim.cpp
+-	encoder/basisu_astc_decomp.cpp
+-	encoder/basisu_uastc_enc.cpp
+-	encoder/basisu_bc7enc.cpp
+-	encoder/lodepng.cpp
+-	encoder/apg_bmp.c
+-	encoder/jpgd.cpp
+-	encoder/basisu_kernels_sse.cpp
+-	transcoder/basisu_transcoder.cpp
++set(BASISU_SRC_LIST ${COMMON_SRC_LIST} transcoder/basisu_transcoder.cpp)
++
++if (NOT TRANSCODER_ONLY)
```

Comment:
This changes behavior of the upstream project.... we typically avoid this here in CCI...

Id love to encourage you to contribute this upstream https://github.com/conan-io/conan-center-index/pull/6468/files#r678224294

Reply:
No, it should build the whole library as before by default. But if possible, I will try to rewrite this CMakeLists.txt so it won't require so many patch changes here and send to upstream project.